examples/record-playback/rs-record-playback.cpp

```diff
@@ -209,6 +212,27 @@ catch (const std::exception& e)
     return EXIT_FAILURE;
 }
 
+std::string bag_file()
+{
+    char sep = '/';
+    std::string tmp_path = std::getenv("TEMP");
+    std::string bag = "a.bag";
+
+    #ifdef _WIN32
```

Comment:
Avoid platform-specific #ifdefs and explicit query of environment vars. 
Replace with existing 3rd-party from [common/os.h](https://github.com/IntelRealSense/librealsense/blob/master/common/os.h#L62)

Then I believe, the rest will be unnecessary

Reply:
@manson, is there any update?
putting @maloel on notice

Reply:
just pushed new commit, Copying here description from commit: 
```rs-record-payback``` now uses existing universal function ```get_folder_path``` from ```common``` folder. Previously it was defined in ```/common/os.cpp``` file dealing with files functions, file dialogs and so on in one file, but direct usage of it led to a conflict with existing Opengl libraries used by examples (and ```rs-record-payback```) as it loads this as well. So files functions (without dialogs) was extracted to a file fs.cpp (and fs.h) and this functions may be used in any part of realsense solution.```os.h``` includes ```fs.h``` as well and any possible usage of this functions by including ```os.h``` leaves it operational (but with Opengl libraries included). ```rs-record-payback``` now directly includes ```fs``` module to get temporary system folder to record to a temporary file.